mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxDAO.java

```diff
@@ -117,7 +119,7 @@ private PreparedStatement prepareRead(Session session) {
             .setUUID(ID, cassandraId.asUuid())
             .setString(NAME, mailbox.getName())
             .setLong(UIDVALIDITY, mailbox.getUidValidity())
-            .setUDTValue(MAILBOX_BASE, mailboxBaseTupleUtil.createMailboxBaseUDT(mailbox.getNamespace(), mailbox.getUser())));
+            .setUDTValue(MAILBOX_BASE, mailboxBaseTupleUtil.createMailboxBaseUDT(mailbox.getNamespace(), Optional.of(mailbox.getUser()))));
```

Comment:
Can we drop the `Optional.of` ? because here we have an `Optional<Optional<Username>>` if i m not mistaken.